larq_compute_engine/core/types.h

```diff
@@ -10,8 +10,6 @@ enum class FilterFormat { Unknown, HWIO, OHWI, OHWI_PACKED };
 // defines the operating dimension
 enum class Axis { RowWise, ColWise };
 
-enum class BitpackOrder { Canonical, Optimized };
-
```

Comment:
Since we're refactoring this anyway, we can maybe also remove the `Axis` parameter, since we only ever use one of them I think. How do you feel about this?

Reply:
Yep, that makes sense to me :+1: